crates/router/src/core/payments/transformers.rs

```diff
@@ -551,6 +551,8 @@ impl<F: Clone> TryFrom<PaymentAdditionalData<'_, F>> for types::PaymentsAuthoriz
                 None
             }
         });
+        let billing = payment_data.address.billing.get_required_value("billing")?;
```

Comment:
```suggestion
        let billing = payment_data.address.get_billing()?;
```

Reply:
`get_billing` is implemented for router_data